R/plot_price.R

```diff
@@ -1,15 +1,13 @@
 #' Plot the price of cryptocurrency
 #'
 #' @param price_df data frame with time and price data.
-#' @param name character name of the cryptocurrency of interest
-#' @param window vector of two dates in character
 #'
 #' @return ggplot object
 #' @export
 #'
 #' @examples
-#' price_df = data.frame('Symbol'=c('BTC-USDT','BTC-USDT'), Date'=c('2022-01-15', '2022-01-16'), 'Close'=c(1.5, 2.5))
-#' plot_price(price_df)
+#' price_df <- data.frame('Symbol'=c('BTC-USDT','BTC-USDT'), Date'=c('2022-01-15', '2022-01-16'), 'Close'=c(1.5, 2.5))
```

Comment:
There is still one missing `'` around Date in `price_df`. 
```suggestion
#' price_df <- data.frame('Symbol'=c('BTC-USDT','BTC-USDT'), 'Date'=c('2022-01-15', '2022-01-16'), 'Close'=c(1.5, 2.5))
```